server.js

```diff
@@ -1219,15 +1217,20 @@ function objectWebServer() {
         }
 
         if ((urlArray[urlArray.length - 2] === 'videos') && urlArray[urlArray.length - 1].split('.').pop() === 'mp4') {
-            urlArray[urlArray.length - 2] = identityFolderName + '/videos';
+            // videoDir differs on mobile due to inability to call mkdir
+            if (!isMobile) {
+                urlArray[urlArray.length - 2] = identityFolderName + '/videos';
+            } else {
+                urlArray.splice(urlArray.length - 2, 1);
+            }
```

Comment:
I acknowledge that this is pretty messy, but I'm struggling to find a cleaner solution given how messy the surrounding implementation is

Reply:
Yeah this function is a bit on the opaque side